README.md

```diff
@@ -1,7 +1,22 @@
-# Adobe Commerce Bold Checkout Integration.
-## Description
-This module replaces the native Adobe Commerce checkout with Bold Checkout.
-## Installation of the Adobe Commerce module
-https://developer.boldcommerce.com/default/guides/platform-integration/adobe-commerce/installation
+# Adobe Commerce Bold Checkout Integration
+
+This extension replaces the built-in Adobe Commerce checkout experience with Bold Checkout.
+
+The Bold Checkout extension supports Adobe Commerce (formerly known as Magento 2) and Magento Open Source stores.
+
+## Install the Bold Checkout extension
+
+Installation instructions depend on how your checkout is hosted. Refer to the developer documentation below.
+
+- [Bold-hosted](https://developer.boldcommerce.com/default/guides/platform-integration/adobe-commerce/installation): Use the pre-built, optimized, three-page checkout experience provided by Bold.
+- [Self-hosted](https://developer.boldcommerce.com/guides/platform-integration/adobe-commerce/self-hosted-checkout): Use a custom checkout for a more customized experience.
+
 ## Documentation
-How to install a module on your Adobe Commerce platform - https://experienceleague.adobe.com/docs/commerce-cloud-service/user-guide/configure-store/b2b-module.html?lang=en
+
+The Adobe Experience League documentation provides general instructions for installing an extension on your store.
+
+- [How to install a module on your Adobe Commerce platform](https://experienceleague.adobe.com/docs/commerce-cloud-service/user-guide/configure-store/b2b-module.html?lang=en)
```

Comment:
Honestly I think it might be confusing to include this, since they should really be following the instructions above! Maybe just remove??